### ISSUE TITLE
learning rate setting

### ISSUE THREAD

**Issue by jianbaba** [Open-Debin/Emotion-FAN#6]
Hello, I have observed that the learning rate settings for different training sets are very different. How do I set the learning rate and other parameters for other training sets?

**Comment by Open-Debin** [comment]
I thought you can just try some learning_rate then select the best one. There is no universal rule, it's case by case. Also, there is no shortcut for trying parameter, just do it.

**Comment by jianbaba** [comment]
> I thought you can just try some learning_rate then select the best one. There is no universal rule, it's case by case. Also, there is no shortcut for trying parameter, just do it.

Thank you for your reply. I followed your open source code to perform experiments on other data sets (the number of video sequences is half of ck, but the total number of frames is twice of ck). After finding several epochs, the training set is accurate The rate has reached 100%, but the validation set is only about 40%. Have you encountered similar problems? How to solve!

**Comment by Open-Debin** [comment]
Sure. It is overfitting? which dataset?

**Comment by jianbaba** [comment]
> Sure. It is overfitting? which dataset?

I think it is the cause of overfitting. This data set was recorded by ourselves, about 10 people, each of them recorded 5 times, a total of seven expressions. Is there too little data?How to solve!

**Comment by Open-Debin** [comment]
Yes, the data set is not diverse enough. I think you can just training classifer. Also, apply dropout.

**Comment by jianbaba** [comment]
> Yes, the data set is not diverse enough. I think you can just training classifer. Also, apply dropout.

thanks

**Comment by Open-Debin** [comment]
@jianbaba 
Merry Christmas, I recently update the Emotion-FAN, new features include data process, environment install, CK+ code, Baseline code, and more detail instructions. Also, you can find the old version directory of Emotion-FAN in the README.md. I hope my new updates can help you greatly. Please see the Emotion-FAN for more details.
